src/containers/SignUp/SignUp.js

```diff
@@ -41,43 +40,47 @@ class SignUp extends Component {
     this.setState({ [name]: value });
   }
   postData(event) {
+    // debugger;
     event.preventDefault();
     const name = this.state.name;
```

Comment:
@Humnaa use es6 syntax here named `Object Destructing`
For more detail refer here: https://dev.to/sarah_chima/object-destructuring-in-es6-3fm

Reply:
I've made changes according to instruction